### ISSUE TITLE
npm install jsdoc failes

### ISSUE THREAD

**Issue by faandi** [verdaccio/verdaccio#289]
#### My reason:

Installing jsdoc via verdaccio registry fails with the following error message:

```
npm install jsdoc
npm ERR! fetch failed https://npm.msgis.net/babylon/-/babylon-7.0.0-beta.0-ranges.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
```

It works if I switch back from my verdaccio registry to the official npmjs registry.

#### Steps to reproduce:

`npm install jsdoc`

####  App Version:	

```
verdaccio --version
2.3.4                        
```

```
npm --version
3.10.10
```
```
node --version
v6.11.1
```

####  Config file:	

```yaml
storage: ./storage 
auth:
  htpasswd:
    file: ./htpasswd
    max_users: 2
uplinks:
  npmjs:
    url: https://registry.npmjs.org/ 
packages:
  '**':
    access: developer
    publish: developer
    proxy: npmjs
url_prefix: https://my.internal.domain
listen:
 - 0.0.0.0:4873
logs:
  - {type: stdout, format: pretty, level: http}
```

#### Additional information:

verdaccio log:

```
....
http <-- 200, user: developer(XX.XX.XX.XX via XX.XX.XX.XX), req: 'GET /babylon', bytes: 0/20543
http <-- 404, user: developer(XX.XX.XX.XX via XX.XX.XX.XX), req: 'GET /babylon/-/babylon-7.0.0-beta.0-ranges.tgz', error: file doesn't exist on uplink
....
```

**Comment by faandi** [comment]
I tried installing jsdoc with a newer npm and node version:

```
$ nvm use v8
Now using node v8.2.1 (npm v5.3.0)
$ npm install jsdoc
npm ERR! code E404
npm ERR! 404 Not Found: babylon@~7.0.0-beta.19

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/XXXX/.npm/_logs/2017-08-11T08_57_37_078Z-debug.log
```

**Comment by faandi** [comment]
It works now. I had issues installing other packages as well. After some troubleshooting running `npm login` did the trick.

**Comment by faandi** [comment]
It works now. I had issues installing other packages as well. After some troubleshooting running `npm login` did the trick.

**Comment by juanpicado** [comment]
It might be related with #285

**Comment by juanpicado** [comment]
It works for me with `verdaccio@2.3.5` Could you upgrade and try again?
```
➜ npm install jsdo
npm WARN npm-test-hello-world@2.0.2 No description
npm WARN npm-test-hello-world@2.0.2 No repository field.

+ jsdo@4.3.12
added 2 packages in 8.96s

➜ npm i babylon@~7.0.0-beta.19
npm WARN npm-test-hello-world@2.0.2 No description
npm WARN npm-test-hello-world@2.0.2 No repository field.

+ babylon@7.0.0-beta.19
added 1 package in 4.269s
```

**Comment by faandi** [comment]
Hi, i just upgraded to `verdaccio@2.3.6` and it works now.
I have also cleared my storage dir to fix #292.
